src/screen/Screen.java

```diff
@@ -114,4 +127,12 @@ public final int getWidth() {
 	public final int getHeight() {
 		return this.height;
 	}
+
+	public void swapBuffers() {
```

Comment:
I think this method name is unnecessary. double buffering is already implemented by renderer